src/xia2/Modules/MultiCrystal/ScaleAndMerge.py

```diff
@@ -127,6 +127,10 @@
     .short_caption = "Resolve indexing ambiguity"
   cosym {
     include scope dials.algorithms.symmetry.cosym.phil_scope
+    relative_length_tolerance = 0.05
```

Comment:
I think it would be useful to add a help string for the two new parameters, something like "a fractional tolerance used to determine whether cells are similar during the cosym symmetry analysis"?

Reply:
I realised Graeme had also made the same comment on dials.correlation_matrix :laughing: I have now made the .help the same across both scripts.